GMG/Scenes/ChordProgress/ChordProgressIntent.swift

```diff
@@ -80,11 +87,15 @@ final class ChordProgressIntent: ChordProgressIntentProtocol {
         self.scorePlayer?.stop()
     }
 
+    func onTapMuteButton(_ isMuted: Bool) {
+        self.scorePlayer?.setPlayerMuted(isMuted)
+    }
+
     func onTapChordCell(_ chordCell: ChordCell) {
         self.scorePlayer?.seek(chordCell: chordCell)
```

Comment:
이렇게 까지 될 지는 모르겠지만 현재 순서대로면 UI가 바뀌기 전에 오디오가 재생될 수도 있을 것 같은데 두 개 순서를 바꿔보시는 건 어떤가요!
```suggestion
        self.model?.selectChordCell(chordCell)
        self.scorePlayer?.seek(chordCell: chordCell)
```

Reply:
@ehdrbdndns 이 부분은 #60 에서 반영하면 될 듯 합니다!